types/rosie/index.d.ts

```diff
@@ -187,9 +210,9 @@ declare namespace rosie {
      * @param {object=} options
      * @return {*}
      */
-    build(attributes: Object, options: Object): Object;
+    build(attributes: { [k in keyof T]?: T[k] }, options?: any): T;
 
-    buildList(size: number, attributes: Object, options: Object): Object[];
+    buildList(size: number, attributes: { [k in keyof T]?: T[k] }, options: any): T[];
```

Comment:
Same goes for here, both `attributes` and `options` are required, whereas they should be optional :)